sliceutil/anonymize.go

```diff
@@ -9,3 +9,13 @@ func Anonymize[T any](src []T) []interface{} {
 
 	return dst
 }
+
+func String[T ~string](src []T) []string {
```

Comment:
missing documentation on a public function

Reply:
This should be called "Strings" or "ToStrings" or "ToStringSlice" or something similar. `.String` means give me a string representation of the object, e.g. `String([]int{1,2,3}) => "[ 1, 2, 3 ]" `